src/dbcp/data_mart/projects.py

```diff
@@ -540,13 +540,172 @@ def create_long_format(engine: sa.engine.Engine) -> pd.DataFrame:
     return long_format
 
 
-def _pudl_eia860m_changelog(engine: sa.engine.Engine) -> pd.DataFrame:
-    """Get the PUDL EIA860M changelog table."""
-    with engine.connect() as con:
-        pudl_eia860m_changelog = pd.read_sql_table(
-            "pudl_eia860m_changelog", con, schema="data_warehouse"
+def get_eia860m_current(
+    engine: sa.engine.Engine, date_as_of: Optional[str] = None
+) -> pd.DataFrame:
+    """Get the most recent EIA860M data."""
```

Comment:
Can you add argument and return value descriptions to the doc strings please.

Reply:
Yes, will do. I forgot to document those because I didn't end up implementing the logic. Maybe I'll just remove it for now.